crates/bevy_gilrs/src/converter.rs

```diff
@@ -1,7 +1,9 @@
-use bevy_input::gamepad::{Gamepad, GamepadAxisType, GamepadButtonType};
+use bevy_input::gamepad::{GamepadAxisType, GamepadButtonType, GamepadId};
 
-pub fn convert_gamepad_id(gamepad_id: gilrs::GamepadId) -> Gamepad {
-    Gamepad::new(gamepad_id.into())
+//  TODO: bevy_input directly depends on Gilrs::GamepadId's. This could cause conflicts with other plugins.
+//  Instead, bevy_input should provide a unique GamepadId and bevy_gilrs keep a mapping between them
```

Comment:
Agreed. I'm fine to leave this for follow-up though.

Reply:
Seems like Entity should be that unique id

Reply:
I like the idea in your edit. Eventually we could use entity disabling instead.